src/pygambit/tests/test_behav.py

```diff
@@ -724,3 +729,27 @@ def test_infoset_belief(self):
             self.profile_rational.belief(self.game.infosets[0].members[0]) ==
             gbt.Rational(1, 1)
         )
+
+    def test_payoff_with_chance_player(self):
+        """Test to ensure that payoff called with the chance player raises a ValueError"""
+        chance_player = self.game_with_chance.players.chance
+        self.assertRaises(ValueError, self.profile_double_with_chance.payoff, chance_player)
+        self.assertRaises(ValueError, self.profile_rational_with_chance.payoff, chance_player)
+
+    def test_infoset_value_with_chance_player_infoset(self):
+        """Test to ensure that infoset_value called with an infoset of the chance player
+        raises a ValueError"""
```

Comment:
Style for triple-quoted strings that are over more than one line usually put the closing triple quote on its own line.